docs/customization/configuration.md

```diff
@@ -51,11 +40,16 @@ INSTALLED_APPS = (
   'django.contrib.humanize',
   'django.contrib.staticfiles',
 
-  'mytheme', # Instead of 'default'
+  'themes.mytheme',
+  'themes.default',
   'core',
 )
 ```
 
+Additional settings are documented in [`onisite/settings_base.py`](https://github.com/open-oni/open-oni/blob/master/onisite/settings_base.py).
```

Comment:
These URLs can be made without the full URL that specifies a particular branch right? Similar to links we're using elsewhere here in the docs that are relative to the branch being viewed. Could you update this? And maybe search for any others like it? I'll comment on them if I see others. :pray: